Randomizer.SMZ3/World.cs

```diff
@@ -118,7 +131,8 @@ class World {
         private void SetRewards(Random rnd) {
             var rewards = new[] {
                 PendantGreen, PendantNonGreen, PendantNonGreen, CrystalRed, CrystalRed,
-                CrystalBlue, CrystalBlue, CrystalBlue, CrystalBlue, CrystalBlue }.Shuffle(rnd);
+                CrystalBlue, CrystalBlue, CrystalBlue, CrystalBlue, CrystalBlue,
+                BossTokenKraid, BossTokenPhantoon, BossTokenDraygon, BossTokenRidley}.Shuffle(rnd);
```

Comment:
I rearranged the medallion and reward setup in my other PR. Please review those changes to identify how this needs to be combined.